src/components/field/index.tsx

```diff
@@ -277,7 +275,10 @@ export interface DraggedFieldIdentifier {
 const fieldSource: DragSourceSpec<FieldProps> = {
   beginDrag(props): DraggedFieldIdentifier {
     const {fieldDef, parentId, schema} = props;
-    const filter = getFilter(fieldDef, schema.domain(fieldDef as FieldQuery));
+    if (isWildcard(fieldDef.field)) {
```

Comment:
Can you reply to explain what is this logic for and also add comment accordingly?